app/models/miq_request_workflow.rb

```diff
@@ -501,6 +501,16 @@ def required_description(dlg, fld)
     "'#{dlg[:description]}/#{fld[:required_description] || fld[:description]}'"
   end
 
+  def allowed_storage_profiles(options = {})
+    model_name = options[:category]
```

Comment:
@jameswnl Is there a reason to get the category from the dialog field?  Will it ever be anything besides `StorageProfile`?  In the `allowed_storage_profiles` method I would think this would be a known value.